controllers/drcluster_controller.go

```diff
@@ -936,51 +936,9 @@ func (u *drclusterInstance) removeFencingCR(cluster ramen.DRCluster) (bool, erro
 	err := u.mwUtil.DeleteManifestWork(fmt.Sprintf(util.ManifestWorkNameFormat,
 		u.object.Name, cluster.Name, util.MWTypeNF), cluster.Name)
 	if err != nil {
-		if errors.IsNotFound(err) {
-			return u.ensureNetworkFenceDeleted(cluster.Name)
```

Comment:
Why u.ensureNetworkFenceDeleted is not needed now?

Reply:
We discussed it with @raghavendra-talur , if Manifest work will be deleted - the networkfence will be deleted too, so it's enough to delete manifest work.